### PR TITLE
[botcom] add animations for file list insertions/removals

### PR DESCRIPTION
**Pull request by ds300**
This is especially important for deletions and duplications. it can be really hard to follow what's going on in the file list without it. you know what they say, 'a lil bit of juice makes the mouth happy'

Using https://auto-animate.formkit.com/ which is small and respects `prefers-reduced-motion`.

https://github.com/user-attachments/assets/10a65601-118c-4d8a-9cfb-6536cc798f56

### Change type

- [x] `other`

### CODE REVIEW

**Comment by ds300** [comment]
eh, not into this today, plus it messes with e2e tests. deletions and duplications will hopefully be rare enough that this shouldn't matter too much.

**Comment by mimecuvalo** [comment]
this looked nice fwiw!
